.github/workflows/build.yaml

```diff
@@ -14,19 +14,25 @@ jobs:
     strategy:
       matrix:
         php:
-        - '^(7\.4)-apache$'
-        - '^(8\.0)-apache$'
-        - '^(7\.4)-fpm$'
-        - '^(8\.0)-fpm$'
+        - '^(7\.4)(\.\d+)?-apache$'
+        - '^(8\.0)(\.\d+)?-apache$'
+        - '^(8\.1)(\.\d+)?-apache$'
+        - '^(7\.4)(\.\d+)?-fpm$'
+        - '^(8\.0)(\.\d+)?-fpm$'
+        - '^(8\.1)(\.\d+)?-fpm$'
     steps:
+      - name: Setup multi-architecture support
+        run: 'docker run --rm --privileged multiarch/qemu-user-static --reset -p yes'

```

Comment:
is dit eigenlijk wel nodig? doet de setup-buildx-action dit niet al?

Reply:
Nope, dan krijg je standaard maar 2 architecturen (te zien in de setup-buildx-action onder 'Inspect'). Deze voegt de emulatoren toe.